partner_communication_switzerland/models/res_partner.py

```diff
@@ -383,3 +385,4 @@ def start_new_donors_onboarding(self):
                 })
 
             partner.onboarding_new_donor_start_date = fields.Date.today()
+            partner.onboarding_new_donor_hash = hashlib.md5(f"{partner.id}".encode()).hexdigest()
```

Comment:
Will this always produce the same hash for a given partner id? Shouldn't we use a random hash like the uuid library provides?

Reply:
You're right the hash would be the same every time (for the same email address)
I can change it